modules/nw-metallb-frr-k8s-configuration-crd.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-metallb-frrconfiguration-crd_{context}"]
+== Configuring the FRRConfiguration CRD
+
+=== Configuring one or more routers
```

Comment:
Would it make sense to describe the api in tabular form, as we do in other parts of the docs?

Reply:
I've added a table at the end of the [Configuring the FRRConfiguration CRD](https://75492--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-frr-k8s.html#nw-metallb-frrconfiguration-crd_configure-metallb-frr-k8s) section